### DISCUSSION TITLE
Save (Upload) files when using a form in a Livewire component

### DISCUSSION THREAD

**Discussion by swissjust-america** [filamentphp/filament#11162] [Help]
### Package

Form builder

### Package Version

3,2

### How can we help you?

In the doc mention to use  **getState()** to validate and get the data form Form, but what about for files? how do I store/upload them to a directory or S3 ?? 
I can found no method for that.

```
use App\Models\Post;
 
public function create(): void
{
    Post::create($this->form->getState());
}
```

any help would be apreciate!
thanks in advance

**Comment by swissjust-america** [comment]
$this->form->getState() does it, but I have some issue with S3 that why it is not upload.

**Comment by LucasSampai0** [comment]
Hi, i'm having the same problem right now, any advance in that question?
